src/main/kotlin/mate/academy/LetterRemover.kt

```diff
@@ -1,5 +1,13 @@
 package mate.academy
 
-class LetterRemover {
-    // implement ...
+object LetterRemover {
```

Comment:
Please check task requirements.
There should be companion object
https://kotlinlang.org/docs/object-declarations.html#companion-objects